Packs/Zoom/Integrations/Zoom/Zoom.yml

```
@@ -1225,6 +1225,15 @@ script:
       name: channel_id
     description: Delete a mirrored Zoom channel.
     name: close-channel
+  - name: zoom-delete-user-token
+    description: Revoke a user's Zoom SSO session if their account becomes compromised. Use the zoom-list-users command to get the user ID or user email.
```

```suggestion
    description: Revoke a user's Zoom SSO session. Use the zoom-list-users command to get the user ID or user email.
```

lets stick to the actual behavior of the command and not to the use case